packages/diagrams/frontend/sirius-components-diagrams/src/renderer/helper-lines/useHelperLines.types.ts

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { NodeChange } from 'reactflow';
+
+export interface UseHelperLinesValue {
+  helperLinesEnabled: boolean;
+  setHelperLinesEnabled: (enable: boolean) => void;
+  verticalHelperLine: number | null;
+  horizontalHelperLine: number | null;
+  computeHelperLines: (changes: NodeChange[]) => NodeChange[];
```

Comment:
Maybe `applyHelperLines` or something like that?
`computeHelperLines` gives the impression that this, well, computes the value of helper lines, while as I understand it it applies the snap constraints implied by the helper lines to "tweak" the node changes.